cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -239,6 +239,8 @@ type NodeConfiguration struct {
 	DiscoveryTokenAPIServers []string
 	// DiscoveryTimeout modifies the discovery timeout
 	DiscoveryTimeout *metav1.Duration
+	// KubeProxy holds configuration for the k8s service proxy.
+	KubeProxy KubeProxy
```

Comment:
Since this is common to both MasterConfiguration and NodeConfiguration, there should probably be a helper method added to the CommonConfiguration interface for this as well.

Reply:
Oh, got it. I'll implement an `GetKubeProxy()` method